packages/react-router-redux/examples/ssr/server-store.js

```diff
@@ -0,0 +1,32 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import { routerMiddleware } from 'react-router-redux';
+import thunk from 'redux-thunk';
+
+// On the server we use MemoryHistory, that's an important distinction
+import createHistory from 'history/createMemoryHistory';
+g;
```

Comment:
um...

Reply:
Whoops!  Getting this fixed - good find.